src/library/Uno.Material/Uno.Material.csproj

```diff
@@ -193,9 +190,12 @@
 
 		<XamlMergeInput Remove="Styles\Controls\v1\ProgressRing.xaml" Condition="'$(UseWinUI)'=='true'" />
 		<XamlMergeInput Remove="Styles\Controls\v2\ProgressRing.xaml" Condition="'$(UseWinUI)'=='true'" />
-
 		<Page Remove="Styles\Controls\v1\ProgressRing.xaml" Condition="'$(UseWinUI)'=='true'" />
 		<Page Remove="Styles\Controls\v2\ProgressRing.xaml" Condition="'$(UseWinUI)'=='true'" />
+		<XamlMergeInput Remove="Styles\Controls\v1\ProgressRingWinUI.xaml" Condition="'$(UseWinUI)'!='true'" />
```

Comment:
just for suggestion, you can simplify this code:
1. create a new ItemGroup like `UwpXaml` and `WinUIXaml` to add ProgressRing.xaml and ...WinUI.xaml:
```xml
<UwpXaml Include="Styles\Controls\v1\ProgressRing.xaml" />
<WinUIXaml Include="Styles\Controls\v1\ProgressRingWinUI.xaml" />
<!-- and, add the v2\ or any other xaml to add/remove conditionally... -->
```

2. and, then you can refactor the everything else as follow:
```xml
<XamlToAdd Include="@(UwpXaml)" Condition="'$(UseWinUI)'!='true'" />
<XamlToAdd Include="@(WinUIXaml)" Condition="'$(UseWinUI)'=='true'" />
<XamlToRemove Include="@(UwpXaml)" Condition="'$(UseWinUI)'=='true'" />
<XamlToRemove Include="@(WinUIXaml)" Condition="'$(UseWinUI)'!='true'" />

<XamlMergeInput Remove="@(XamlToAdd)" />
<Page Remove="@(XamlToRemove)" />
```